qa/tasks/vstart_runner.py

```diff
@@ -599,7 +611,7 @@ def _asok_path(self):
 
     def mount(self, mount_path=None, mount_fs_name=None, mount_options=[], **kwargs):
         self.setupfs(name=mount_fs_name)
-        if opt_use_ns:
```

Comment:
It doesn't feel good to reject the flag passed by user silently. Perhaps, we should print a message saying that flag was silently rejected.

Or better yet, let's terminate the execution and ask the user to either pass --teardown (so that vstart_runner.py can setup new cluster which it always does without using localhost) or to take away --usens from command arguments.

Reply:
Hmm, yeah this makes sense. And there already has one message in Line#556, is this enough ?